script/boxen

```diff
@@ -17,7 +18,15 @@ end
 
 # Yeah yeah, I like to be explicit.
 
-at_exit { myself.flock File::LOCK_UN }
+at_exit {
```

Comment:
At the end of the file, the function "exit Boxen::CLI.run ARGV" starts the boxen process. When this task has completed, whatever is inside of the "at_exit" brackets gets called. The code that I wrote pulls the latest changes for the node-build repo (which stores the updated list of node versions) and refreshes the environmental variables. The node version it will update to is located in the "node_version.rb" file I made in the config directory, and can be changed to whatever version is desired. If that specified version is not already installed (in /opt/boxen/nodenv/versions), then it will install it and update the global version of node. 

The myself.flock File::LOCK_UN line was in before, so I left it at the end.